packages/btc/src/base-transfer.ts

```diff
@@ -1,46 +1,54 @@
-/* eslint-disable @typescript-eslint/no-unused-vars */
-
-import { BitcoinResource } from "@buildwithsygma/core/src";
-import { Config } from "@buildwithsygma/core/types";
-
-type BaseTransferParams = {
-  destinationAddress: string;
-  amount: bigint;
-};
-
-type BitcoinTransferRequest = {
-  destinationAddress: string;
-  amount: bigint;
-  depositAddress: string;
-};
-
-export async function createBitcoinTransferRequest(params: BaseTransferParams): Promise<BaseTransfer> {
-  throw new Error("Method not implemented");
-}
+import type { BitcoinResource } from '@buildwithsygma/core/src';
+import type { BitcoinConfig, Config, Domain } from '@buildwithsygma/core/types';
+import type { BaseTransferParams } from 'types';
 
 export abstract class BaseTransfer {
-  constructor(transfer: BaseTransferParams, config: Config) {}
+  protected destinationAddress: string;
+  protected amount: number;
```

Comment:
`bigint`

Reply:
also the smallest unit on btc is a satoshi which is just `1e-8`

Reply:
for the sake of consistency `BigInt` would be better IMO as we have it in all other packages

@Lykhoyda @MakMuftic wdyt?

Reply:
Yep, I agree, it would be more consistent to use `BigInt` here

Reply:
I agree as well with Mak